### PR TITLE
add telemetry and faq, fix broken links

### PR DESCRIPTION
**Pull request by FeynmanZhou**
Signed-off-by: FeynmanZhou <pengfeizhou@yunify.com>

### CODE REVIEW

File: content/en/docs/faq/telemetry.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+---
+title: "Telemetry in KubeSphere"
+keywords: "Installer, Telemetry, KubeSphere, KubeSphere"
+description: "Telemetry collects aggregate information of the KubeSphere installation."
+
+Weight: 7100
+---
+
+Telemetry collects aggregate information about the size of KubeSphere Cluster installations, KubeSphere and Kubernetes versions, Cluster size, the running time of cluster, error logs, etc. We promise that this information is only used by KubeSphere Community to improve the product and we will not shared with any third-parties.
```
Comment (Sherlock113) [inline content/en/docs/faq/telemetry.md:9]: ->
Telemetry collects aggregate information about the size of KubeSphere cluster installations, KubeSphere and Kubernetes versions, cluster size, cluster running time, error logs, etc. We promise that the information is only used by KubeSphere Community to improve products and will not be shared with any third parties.

File: content/en/docs/faq/console-faq.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+title: "Questions about KubeSphere Console"
+keywords: "FAQ, console, KubeSphere, Kubernetes"
+description: "FAQ is designed to answer and summarize the questions our users most frequently ask about KubeSphere Console."
+
+Weight: 7100
+---
+
+**What kind of browsers does KubeSphere support?**
+
+In general, KubeSphere console supports some popular browsers including **Chrome, Firefox, Safari, Opera, and Edge.** You only need to consider the supported versions of these browsers listed in the green box of the table below:
```
Comment (Sherlock113) [inline content/en/docs/faq/console-faq.md:11]: KubeSphere console supports some popular browsers ->
KubeSphere console supports major web browsers

File: content/en/docs/faq/_index.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+description: "FAQ is designed to answer and summarize the questions our users most frequently ask about KubeSphere."
+layout: "single"
+
+linkTitle: "FAQ"
+weight: 7000
+
+icon: "/images/docs/docs.svg"
+---
+
+FAQ is designed to answer and summarize the questions our users most frequently ask about KubeSphere. This section is in a long-term maintenance for v3.0.x.
```
Comment (Sherlock113) [inline content/en/docs/faq/_index.md:12]: ->
FAQ is designed to answer and summarize the questions users ask most frequently about KubeSphere. This section will be updated regularly for v3.0.x.

File: content/en/docs/faq/_index.md [hunk 4]
```diff
@@ -0,0 +1,12 @@
+---
+title: "FAQ"
+description: "FAQ is designed to answer and summarize the questions our users most frequently ask about KubeSphere."
```
Comment (Sherlock113) [inline content/en/docs/faq/_index.md:3]: ->
FAQ is designed to answer and summarize the questions users ask most frequently about KubeSphere.

**Review by Sherlock113 — COMMENTED** [review]
Please have a look at my comments. Thanks.

**Comment by FeynmanZhou** [comment]
Fixed

**Pull request merged**